desktop-widgets/preferences/preferences_units.ui

```diff
@@ -234,7 +234,7 @@
    <item>
     <widget class="QGroupBox">
      <property name="title">
-      <string>Duration units</string>
+      <string>Duration units in dive list table</string>
```

Comment:
Hmm - are you sure? We use the same helper function most everywhere. E.g. in the divecomputer import and the dive info.